### ISSUE TITLE
Bubble Bash Regression

### ISSUE THREAD

**Issue by thundergun1** [hikari-no-yume/touchHLE#295]
I've spotted another regression in the post-Dungeon Hunter nightly builds, this time for Bubble Bash. 

In version 0.2.2, the game works perfectly with the corrupted graphics from previous versions finally fixed, but in the newest nightly builds the visual corruption has returned.

![Screenshot_20240504_084833_touchHLE](https://github.com/touchHLE/touchHLE/assets/133943803/711220f4-67ca-4a4a-bd97-4561bd93cec1)

With it not being a crash i'm assuming a log won't help but if you need anything extra just let me know.

**Comment by ciciplusplus** [comment]
Is it only on Android on your side?

I don't see regressions on Windows build for example.
![Capture3](https://github.com/touchHLE/touchHLE/assets/91161081/5fa82016-30b8-480a-9caa-bbf9e3ee7ae3)

**Comment by thundergun1** [comment]
Yeah it's on Android, I would have chalked it up to the usual Adreno issues I have but with it working fine on 0.2.2 I figured that wasn't the case

**Comment by ciciplusplus** [comment]
Do you know which commit (== version of a nightly build) in particular introduced a regression for you?

**Comment by thundergun1** [comment]
I currently have v0.2.2-33-g37b8a04 installed and it was on this build i discovered the regression

https://github.com/touchHLE/touchHLE/commit/37b8a04b7f2aba529477a2aed8ff745156f37126

**Comment by hikari-no-yume** [comment]
For what it's worth, I noticed that it's seemingly random whether a report about this game in the app DB or elsewhere has the corruption or not. I haven't tested it myself, but I wonder if it is actually random, or more precisely based on some environmental factor that's hard to pinpoint.

**Comment by thundergun1** [comment]
Forgot to mention on here that since upgrading to a new phone (with a Mali GPU rather than Adreno) I no longer have this issue, so it definitely seems to be an Adreno issue, just confused as to why the graphics were fine for a little bit before corrupting again.
